x-pack/legacy/plugins/uptime/public/lib/helper/render_with_router.tsx

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Let's delete this file as we have `./helper_with_router.ts` in this same directory now.

Reply:
must have messed up during conflicts.